FAWE_Paster/src/main/java/de/butzlabben/missilewars/missile/paste/v1_20/fawe/FAWE_Paster.java

```diff
@@ -74,18 +74,15 @@ public void pasteSchematic(File schematic, Vector locationVec, int rotation, org
         ClipboardFormat clipboardFormat = ClipboardFormats.findByFile(schematic);
         
         try (ClipboardReader clipboardReader = clipboardFormat.getReader(new FileInputStream(schematic));
-             var session = WorldEdit.getInstance().newEditSession(weWorld)) {
-            
-            ClipboardHolder clipboardHolder = new ClipboardHolder(clipboardReader.read());
-            clipboardHolder.setTransform(new AffineTransform().rotateY(rotation));
-            
-            Operation pasteBuilder = clipboardHolder
-                    .createPaste(session)
-                    .to(blockVec)
-                    .ignoreAirBlocks(true)
-                    .build();
-            Operations.completeBlindly(pasteBuilder);
-            
```

Comment:
According to my information, it should work just as well. Did you have problems with it?